pkg/virt-controller/watch/migration_test.go

```diff
@@ -91,7 +91,7 @@ var _ = Describe("Migration watcher", func() {
 
 	shouldExpectMigrationFinalizerRemoval := func(migration *virtv1.VirtualMachineInstanceMigration) {
 		migrationInterface.EXPECT().Update(gomock.Any()).Do(func(arg interface{}) (interface{}, interface{}) {
-			Expect(len(arg.(*virtv1.VirtualMachineInstanceMigration).Finalizers)).To(Equal(0))
+			Expect(arg.(*virtv1.VirtualMachineInstanceMigration).Finalizers).To(HaveLen(0))
```

Comment:
```suggestion
			Expect(arg.(*virtv1.VirtualMachineInstanceMigration).Finalizers).To(BeEmpty())
```
I'm told that this is more idiomatic.

Reply:
Yup, this is right, thanks :)